substrate/frame/src/lib.rs

```diff
@@ -213,19 +213,25 @@ pub mod prelude {
 	#[doc(no_inline)]
 	pub use super::derive::*;
 
-	/// All hashing related things
+	/// All hashing related things.
 	pub use super::hashing::*;
 
+	/// All arithmetic types used for safe math.
+	pub use super::arithmetic::*;
+
 	/// Runtime traits
 	#[doc(no_inline)]
 	pub use sp_runtime::traits::{
-		Bounded, DispatchInfoOf, Dispatchable, SaturatedConversion, Saturating, StaticLookup,
-		TrailingZeroInput,
+		Bounded, DispatchInfoOf, Dispatchable, OpaqueKeys, SaturatedConversion, Saturating,
+		StaticLookup, TrailingZeroInput, ValidateUnsigned,
 	};
 
-	/// Other error/result types for runtime
+	/// Other error/result types for runtime.
```

Comment:
This comment is not correct anymore, you need to change it or add new items in a separate line.

Reply:
updated the doc